src/on-behalf-of/use-on-behalf-of.tsx

```diff
@@ -0,0 +1,13 @@
+import {useRemoteConfig} from '@atb/RemoteConfigContext';
+import {useDebugOverride} from '@atb/debug';
+import {StorageModelKeysEnum} from '@atb/storage';
+
+export const useOnBehalfOf = () => {
+  const {enable_on_behalf_of} = useRemoteConfig();
+  const [debugOverride] = useOnBehalfOfEnabledDebugOverride();
```

Comment:
Note: we may also want to return `debugOverrideReady` here, but we can wait to add it until it is confirmed to be needed. Generally it should be used when API calls are going to depend on the flag.